builder/builder.mk

```diff
@@ -38,13 +38,16 @@ builder/docker/swift/dev/publish:
 	docker buildx build . -f builder/docker/swift/Dockerfile --platform linux/amd64,linux/arm64 -t suborbital/builder-swift:dev --push
 
 # tinygo docker targets
-builder/docker/tinygo:
+builder/docker/tinygo-builder:
+	docker build . -f builder/docker/tinygo/Dockerfile.builder -t suborbital/builder-tinygo-builder:$(ver)
+
+builder/docker/tinygo: builder/docker/tinygo-builder
```

Comment:
I don't particularly want to build the base image every time... could we instead tag the base image based on the version of TinyGo, and just push it to Dockerhub?